contrastive_training/contrastive_main.py

```diff
@@ -128,11 +129,12 @@
 #training loop
 #pdb.set_trace()
 if use_subsample==1:
-    cmd = "mkdir -p ./contrastive_vali_figures_"+str(training_size)+"_lmda_"+lmda_str+"_beta_"+beta_str+"_mode_"+args.mode+"_temp_"+str(args.temperature)+"_wdecay_"+str(args.weightdecay)+"_subsample"
+    cmd = "mkdir -p .\contrastive_vali_figures_"+str(training_size)+"_lmda_"+lmda_str+"_beta_"+beta_str+"_mode_"+args.mode+"_temp_"+str(args.temperature)+"_wdecay_"+str(args.weightdecay)+"_subsample"
```

Comment:
Please run things on the Ocean machines. Do not add Windows' specific notation in the code.

Ideally the code could work on any platform, but Windows is not a requirement. It is not used on the back-end (server, cloud, etc.). Your code MUST work for UNIX/Linux.

Reply:
Contrastive part has not been generalized, just a test version on local machine.